content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -110,6 +111,29 @@ either `Delete` or `Retain`. If no `reclaimPolicy` is specified when a
 Persistent Volumes that are created manually and managed via a storage class will have
 whatever reclaim policy they were assigned at creation.
 
+### Allow Volume Expansion
+
+{{< feature-state for_k8s_version="v1.15" state="beta" >}}
```

Comment:
expansion was beta in 1.11

Reply:
Thank you @msau42 . I have corrected the version